app/src/main/java/org/simple/clinic/home/overdue/OverdueAppointmentNew.kt

```diff
@@ -0,0 +1,130 @@
+package org.simple.clinic.home.overdue
+
+import android.os.Parcelable
+import androidx.room.Dao
+import androidx.room.Embedded
+import androidx.room.Query
+import kotlinx.parcelize.Parcelize
+import org.simple.clinic.overdue.Appointment
+import org.simple.clinic.overdue.callresult.CallResult
+import org.simple.clinic.patient.Gender
+import org.simple.clinic.patient.PatientAgeDetails
+import org.simple.clinic.patient.PatientPhoneNumber
+import java.time.LocalDate
+import java.util.UUID
+
+@Parcelize
+data class OverdueAppointmentNew(
```

Comment:
Is there a reason why we did we not go with `OverdueAppointment_Old` and `OverdueAppointment` names for this like we usually do when we are creating a new version of something?